engines/bops_core/app/assets/stylesheets/bops_core/components/_task_accordion.scss

```
@@ -178,6 +178,18 @@
 
     & > :last-child {
       margin-bottom: 0;
+
+      & > :last-child {
+        margin-bottom: 0;
+
+        & > :last-child {
+          margin-bottom: 0;
+
+          & > :last-child {
+            margin-bottom: 0;
+          }
+        }
+      }
     }
   }
 }
```

Would this work with just one layer of nesting but without the immediate-child operator?

```suggestion
    & :last-child {
      margin-bottom: 0;
    }
}
```